src/scripts/admin/admin.store.js

```diff
@@ -90,6 +91,45 @@ let UserStore = Reflux.createStore({
     },
 
     /**
+     * Search username and email
+     *
+     */
+    searchUsername(searchInput){
```

Comment:
I would rename this to something like 'searchUser' since it searches on usernames and emails.